src/BinFile/PETypes.fs

```diff
@@ -28,6 +28,37 @@ namespace B2R2.BinFile.PE
 
 open B2R2
 open System.Reflection.PortableExecutable
+open System.Reflection.Metadata
+
+/// Base Relocation Type.
+type BaseRelocType =                                                                                              
+  | IMAGE_REL_BASED_ABSOLUTE        = 0
+  | IMAGE_REL_BASED_HIGH            = 1
+  | IMAGE_REL_BASED_LOW             = 2
+  | IMAGE_REL_BASED_HIGHLOW         = 3
+  | IMAGE_REL_BASED_HIGHADJ         = 4
+  | IMAGE_REL_BASED_MIPS_JMPADDR    = 5
+  | IMAGE_REL_BASED_ARM_MOV32       = 5
+  | IMAGE_REL_BASED_RISCV_HIGH20    = 5
+  | Reserved                        = 6
+  | IMAGE_REL_BASED_THUMB_MOV32     = 7
+  | IMAGE_REL_BASED_RISCV_LOW12I    = 7
+  | IMAGE_REL_BASED_RISCV_LOW12S    = 8
+  | IMAGE_REL_BASED_MIPS_JMPADDR16  = 9
+  | IMAGE_REL_BASED_DIR64           = 10
```

Comment:
`BaseRelocType` has underscores (_) in their union names, but this is not a preferred style in .NET. See https://fsharp.org/specs/component-design-guidelines/#-avoid-using-underscores-in-names, and https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md for your reference.

Can you change these to use PascalCase instead? For example, `ImageRelBasedAbsolute` instead of `IMAGE_REL_BASED_ABSOLUTE`.

Reply:
To be honest I was neither happy with this style, I'll modify it :)